profile/README.md

```diff
@@ -1,36 +1,28 @@
 
 ### Our Open Source Solutions
-[10up's Open Source Practice](https://10up.com/blog/2019/open-source-practice/) is committed to the cultivation of a vibrant and healthy open source ecosystem. Our own open projects are often inspired by recurring needs and patterns we see across 10up clients, including:
+Our Open Source Practice is committed to the cultivation of a vibrant and healthy open source ecosystem. Our own open projects for WordPress are often inspired by recurring needs and patterns we see across our clients, including:
 * [Distributor](https://github.com/10up/distributor), a WordPress plugin that makes it easy to syndicate and reuse content across your websites.
 * [ElasticPress](https://github.com/10up/ElasticPress), the only fully-integrated solution for adding the power of Elasticsearch to a WordPress site.
 * [ClassifAI](https://github.com/10up/classifai), a WordPress plugin that leverages cloud-based services like IBM Watson and Microsoft Azure AI to augment WordPress-powered websites with artificial intelligence and machine learning technology.
 
-Our full complement of over 100 open source solutions are [available on GitHub](https://github.com/orgs/10up/repositories).
+Our full complement of over 100 open source solutions that integrate with WordPress are [available on GitHub](https://github.com/orgs/10up/repositories).
 
 ### Our Best Practices
-As a company, 10up strives to build websites, tools, and experiences that demonstrate the highest caliber of craftsmanship. Standardizing our tools and approaches enables us to consistently meet that standard. We share our best practices with the community to empower engineers around the world to benefit from our decade of experience solving some of the most challenging problems on the web.
+As the WordPress and open source experts at Fueled, 10up strives to build websites, tools, and experiences that demonstrate the highest caliber of craftsmanship. Standardizing our tools and approaches enables us to consistently meet that standard. We share our best practices with the community to empower engineers around the world to benefit from our decade of experience solving some of the most challenging problems on the web.
 
 * [10up's Engineering Best Practices](https://10up.github.io/Engineering-Best-Practices/)
 * [10up's Open Source Best Practices](https://10up.github.io/Open-Source-Best-Practices/)
 
 ### Get In Touch
-<p align="center">
-<a href="http://10up.com/contact/"><img src="https://10up.com/uploads/2016/10/10up-Github-Banner.png" width="850"></a>
-</p>
\ No newline at end of file
+
+<a href="http://10up.com/contact/"><img src="https://fueled.com/wp-content/uploads/2025/06/10up-github-banner.webp" alt="Work with the 10up WordPress Practice at Fueled"></a>
```

Comment:
Do we want this to use markdown instead of HTML? This works fine though my code editor always flags this as being wrong. Could replace with:

```suggestion
[![Work with the 10up WordPress Practice at Fueled](https://fueled.com/wp-content/uploads/2025/06/10up-github-banner.webp)](http://10up.com/contact/)
```